flink-runtime/src/main/java/org/apache/flink/runtime/state/v2/RegisteredKeyValueStateBackendMetaInfo.java

```diff
@@ -138,6 +138,13 @@ public TypeSerializerSchemaCompatibility<S> updateStateSerializer(
         return stateSerializerProvider.registerNewSerializerForRestoredState(newStateSerializer);
     }
 
+    @Nonnull
+    public TypeSerializerSchemaCompatibility<N> updateNamespaceSerializer(
+            TypeSerializer<N> newNamespaceSerializer) {
+        return namespaceSerializerProvider.registerNewSerializerForRestoredState(
```

Comment:
nit: is the New in the method name required?  
registerNewSerializerForRestoredState -> registerSerializerForRestoredState

Reply:
Well this is the name of existing interface, we'd better keep this in current PR.